src/org/opendatakit/briefcase/ui/export/components/ExportFormsTableView.java

```diff
@@ -86,14 +82,6 @@
     sorter.sort();
   }
 
-  private Dimension getHeaderDimension(String header) {
```

Comment:
Good idea to extract this into the `TableCustomizer` class.

Reply:
Thank you.

Reply:
Hey, he was praising *me*. 😀